submissions/urls.py

```diff
@@ -0,0 +1,15 @@
+"""
+Submission URLs.
+"""
+
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+
+from submissions.views.xqueue import XQueueViewSet
+
+router = DefaultRouter()
+router.register(r'', XQueueViewSet, basename='xqueue')
+
+urlpatterns = [
+    path('', include(router.urls)),
+]
```

Comment:
Question: Does it have to be at the root like this in order to be backwards compatible with xqueue-watcher expectations?